WordPress/src/main/res/values/strings.xml

```diff
@@ -986,6 +986,27 @@
     <string name="editor_post_settings_set_featured_image">Set Featured Image</string>
 
 
+    <!-- Aztec -->
+    <string name="dialog_link_title">Insert link</string>
```

Comment:
Yep.  I know the string "Insert link" is used for the dialog title, but apparently it's [defined in the library](https://github.com/wordpress-mobile/AztecEditor-Android/blob/develop/aztec/src/main/kotlin/org/wordpress/aztec/AztecText.kt#L976-L993) and [only available in English](https://github.com/wordpress-mobile/AztecEditor-Android/blob/develop/aztec/src/main/res/values/strings.xml#L4-L8).

I removed the string resource in 1d06b97.